libcodechecker/server/api/report_server.py

```diff
@@ -1138,12 +1138,27 @@ def _setReviewStatus(self, report_id, status, message, session):
             raise shared.ttypes.RequestFailed(
                 shared.ttypes.ErrorCode.DATABASE, msg)
 
+    @exc_to_thrift_reqfail
+    @timeit
+    def disableReviewStatusChange(self):
```

Comment:
The name of this function was a little bit misleading because this API function only returns True if review status change is disabled and doesn't disable the review status change actually. So I renamed this function to `reviewStatusChangeIsDisabled`. This is more understandable what it does.